tests/scripts/stderr1.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+## test for issue 2561
+
+OUTPUT=$(mktemp)
+BAT=bat
+code=$($BAT /tmp 2> $OUTPUT; cat $OUTPUT | grep error; echo $?)
+
+if [[ $code == 1 ]]; then
+    echo "stderr test fsil"
+    exit 1
+fi
+
+exit 0
```

Comment:
Can we please turn this into a Rust-based integration test. See `tests/integration_tests.rs` and search for `.stderr` to see some examples.

Reply:
Will do.

Reply:
This script is still here. Can it be removed?